src/webui/www/private/scripts/client.js

```diff
@@ -1656,6 +1661,25 @@ window.addEventListener("load", () => {
     window.qBittorrent.Cache.preferences.init();
     window.qBittorrent.Cache.qbtVersion.init();
 
+    // Setup color scheme switching
+    const firstRun = true;
+    const updateColorScheme = () => {
+        const checkbox = $("colorThemeCheckbox");
+        const colorScheme = LocalPreferences.get("colorScheme");
+        if (colorScheme != null) {
```

Comment:
The linter barfs here. The use of `!=` is intentional because it checks for both `null` and `undefined`.
Is there another preferred way to do it? (I am a JS noob)

Reply:
The rest of the codebase does it explicitly, i.e.
```js
value !== null && value !== undefined
```
It is not necessary in this particular case though. You could simply go with
```js
if (colorScheme) {
```